user/user-guide.md

```diff
@@ -532,6 +534,7 @@ Expression based sorting is not implemented.
 `Optional`
 
 Option to request specific consistency. Options are:
+
```

Comment:
Leading newlines fix list (and code block) rendering

Reply:
glad it was as simple as that